From 048cf525e749bfd370f136ddbf1a631e54e3fd05 Mon Sep 17 00:00:00 2001
From: David Pearce <dpearce@nvidia.com>
Date: Tue, 23 Jul 2013 20:53:45 -0400
Subject: [PATCH 4/4] Revise integrations from nv-jb-mr1 to remove content
 already provided in a change that came with changes in
 android/platform/hardware/libhardware.

Change-Id: Id3e2176a2374cf43ff4ef0b1634031dd4ab41a3f
---
 include/powerservice/IPowerService.h | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/include/powerservice/IPowerService.h b/include/powerservice/IPowerService.h
index 2003ac6..b3de782 100644
--- a/include/powerservice/IPowerService.h
+++ b/include/powerservice/IPowerService.h
@@ -28,16 +28,16 @@
 #include <hardware/power.h>
 #include <powerservice/IPowerServiceClient.h>
 
-typedef enum {
-    APP_PROFILE_CPU_SCALING_MIN_FREQ,
-    APP_PROFILE_CPU_CORE_BIAS,
-    APP_PROFILE_CPU_MAX_NORMAL_FREQ,
-    APP_PROFILE_GPU_CBUS_CAP_LEVEL,
-    APP_PROFILE_GPU_SCALING,
-    APP_PROFILE_PBC_POWER,
-    APP_PROFILE_FAN_CAP,
-    APP_PROFILE_COUNT,
-} app_profile_knob;
+/* typedef enum { */
+/*     APP_PROFILE_CPU_SCALING_MIN_FREQ, */
+/*     APP_PROFILE_CPU_CORE_BIAS, */
+/*     APP_PROFILE_CPU_MAX_NORMAL_FREQ, */
+/*     APP_PROFILE_GPU_CBUS_CAP_LEVEL, */
+/*     APP_PROFILE_GPU_SCALING, */
+/*     APP_PROFILE_PBC_POWER, */
+/*     APP_PROFILE_FAN_CAP, */
+/*     APP_PROFILE_COUNT, */
+/* } app_profile_knob; */
 
 namespace android {
 
-- 
1.8.1.5

